aspnetcore/blazor/state-management/browser-storage.md

```diff
@@ -612,103 +505,6 @@ To persist many different state objects and consume different subsets of objects
 
 :::zone pivot="webassembly"
```

Comment:
I don't think we actually need this pivot for Blazor WebAssembly. Most of the content for Blazor WebAssembly is just documenting the browser platform storage features, which we don't provide any built-in framework support for.